### PR TITLE
move OO tutorial to get-started

### LINKED ISSUE #18137: OO tutorial should move to get started
This content should be updated and serve as the final OO tutorial in the "introduction to programming with C#" section.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ece1648c-e8a7-153d-a22b-350a619aa26d
* Version Independent ID: a0c9221a-cc55-4c31-4f54-0e5fbdcbce25
* Content: [Object-Oriented Programming (C#)](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/object-oriented-programming)
* Content Source: [docs/csharp/programming-guide/concepts/object-oriented-programming.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/concepts/object-oriented-programming.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by BillWagner**
Move the introduction to object oriented programming to the "learn C#" section of tutorials.

Fixes #18137

[Internal preview link](https://review.docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/object-oriented-programming?branch=pr-en-us-20651)

### CODE REVIEW

File: docs/csharp/tutorials/intro-to-csharp/object-oriented-programming.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+---
+title: "Object-Oriented Programming (C#)"
+description: C# provides full support for object-oriented programming including abstraction, encapsulation, inheritance, and polymorphism.
+ms.date: 09/30/2020
+---
+# Object-Oriented programming (C#)
+
+C# is, at its core, an object-oriented language. Four of the key techniques used in object-oriented programming are:
```
Comment (tdykstra) [inline docs/csharp/tutorials/intro-to-csharp/object-oriented-programming.md:8]:
```suggestion
C# is an object-oriented language. Four of the key techniques used in object-oriented programming are:
```

File: docs/csharp/tutorials/intro-to-csharp/object-oriented-programming.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+## Summary
+
+This tutorial demonstrated many of the techniques used in Object-Oriented programming:
+
+- You used *Abstraction* when you kept many details `private` in each class.
+- You used *Encapsulation* when you defined classes for each of the different account types. Those classes described the behavior for that type of account.
+- You used *Inheritance* when you leveraged the implementation already created in the `BankAccount` class to save code.
+- You used *Polymorphism* when you created `virtual` methods that derived classes could override to create specific behavior for that account type.
+
+Congratulations, you've finished all our introduction to C# tutorials. If you're eager to learn more, try more of our [tutorials](../index.md).
```
Comment (tdykstra) [inline docs/csharp/tutorials/intro-to-csharp/object-oriented-programming.md:178]:
```suggestion
Congratulations, you've finished all our introduction to C# tutorials. To learn more, try more of our [tutorials](../index.md).
```

**Review by tdykstra — APPROVED** [review]
LGTM

File: docs/csharp/tutorials/intro-to-csharp/object-oriented-programming.md (hunk 2 above)
Comment (adegeo) [reply docs/csharp/tutorials/intro-to-csharp/object-oriented-programming.md:178]: and
```suggestion
Congratulations, you've finished all of our introduction to C# tutorials. If you're eager to learn more, try more of our [tutorials](../index.md).
```

**Pull request merged**